src/components/Icons.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="icon"
+    v-html="require(`../../svg/${name}.svg`)"
+    :style="{'height': size+'em','width':  size+'em',}"
+  />
+</template>
+
+<script>
+export default {
+  props: {
```

Comment:
It will be good if you add the component name here. eg name: "EosIcon" or something which we can use while importing this component.

Reply:
Thanks a lot for checking out this PR and providing all the suggestions. I will push a PR for the same. @bisht-richa